Resources/views/Form/jquery_layout.html.twig

```diff
@@ -656,6 +656,10 @@
     <script type="text/javascript">
     jQuery(document).ready(function($) {
         $field = $('#{{ id }}');
+        
+        {% if configs.placeholder is defined %}
+            {% set configs = {'placeholder': configs.placeholder|trans} + configs %}
```

Comment:
```twig
{% set configs.placeholder = configs.placeholder|trans %}
```
would be better optmized.
Also a possibility to fetch the translation domain from the form config itself